Content.Client/Store/Ui/StoreMenu.xaml.cs

```diff
@@ -43,24 +43,22 @@ public void UpdateBalance(Dictionary<string, FixedPoint2> balance)
     {
         Balance = balance;
 
-        var currency = new Dictionary<(string, FixedPoint2), CurrencyPrototype>();
-        foreach (var type in balance)
-        {
-            currency.Add((type.Key, type.Value), _prototypeManager.Index<CurrencyPrototype>(type.Key));
-        }
+        var currency = balance.ToDictionary(type =>
+            (type.Key, type.Value), type => _prototypeManager.Index<CurrencyPrototype>(type.Key));
 
         var balanceStr = string.Empty;
-        foreach (var type in currency)
+        foreach (var ((type, amount),proto) in currency)
         {
-            balanceStr += $"{Loc.GetString(type.Value.BalanceDisplay, ("amount", type.Key.Item2))}\n";
+            //this display is fixed to 1 for aesthetic purposes
+            balanceStr += $"{Loc.GetString(proto.DisplayName, ("amount", 1))}: {amount}\n";
```

Comment:
why is the `: {amount}` not just part of the loc string

Reply:
the idea is that the small part of the currency that is used for display is used multiple times across the ui (balance, price, withdraw menu). Instead of creating loc strings for each instance of it within the ui, i just switched it to the short name itself. I thought this would be an easier solution to the problem. I suppose that you could have a loc string for `{$currency}: {$amount}` but it felt really basic so i didn't include it.

Reply:
just have a loc string for that

Reply:
or something cause that feels jank